### DISCUSSION TITLE
１

### DISCUSSION THREAD

**Discussion by yunyingfeng** [rimeinn/rime-moran#101] [Q&A]
怎么改成１８键布局。

**Comment by ksqsf** [comment]
可参考 https://github.com/ksqsf/rime-moran/wiki/%E5%AE%9A%E8%A3%BD%E6%80%9D%E8%B7%AF#%E5%8D%81%E4%B9%9D%E9%8D%B5%E5%AE%9A%E8%A3%BD%E6%96%B9%E6%B3%95

这个仓配置有点老了，可能需要你自己做一些修改。成功后欢迎分享！
